app.rb

```diff
@@ -115,7 +128,12 @@
 
 get '/download/td_agent' do
   @title = "Download Treasure Agent"
```

Comment:
Do we need this `@title` definition?

After redirecting, `@title` is overwritten to `Download Fluent Package` after all.
So this definition looks meaningless to me.

Reply:
No need to define anymore, thanks!